src/neo/SmartContract/Native/NeoToken.cs

```diff
@@ -56,6 +57,61 @@ public sealed class NeoToken : FungibleToken<NeoToken.NeoAccountState>
         internal NeoToken()
         {
             this.TotalAmount = 100000000 * Factor;
+
+            var events = new List<ContractEventDescriptor>(Manifest.Abi.Events)
+            {
+                new ContractEventDescriptor
+                {
+                    Name = "CandidateStateChanged",
+                    Parameters = new ContractParameterDefinition[]
+                    {
+                        new ContractParameterDefinition()
+                        {
+                            Name = "pubkey",
+                            Type = ContractParameterType.PublicKey
+                        },
+                        new ContractParameterDefinition()
+                        {
+                            Name = "registered",
+                            Type = ContractParameterType.Boolean
+                        },
+                        new ContractParameterDefinition()
+                        {
+                            Name = "votes",
+                            Type = ContractParameterType.Integer
+                        }
+                    }
+                },
+                new ContractEventDescriptor
+                {
+                    Name = "Vote",
```

Comment:
How about

```c#
public event void Vote(Hash160 account, Hash160 from, Hash160 to, Integer amount);
```

`from` is the old value of `VoteTo`. `to` is the new value of `VoteTo`. And `amount` is the balance of the account.

Reply:
From and to as ECPoint, isn't it?